service/app/controllers/BaseController.java

```diff
@@ -429,6 +431,23 @@ public Result createCommonResponse(Object response, String key, Request request)
      */
   }
 
+  /**
+   *
+   * @param response
+   * @return
+   */
+  //TODO: Complete the implementation
+  public Result createCommonResponse(Object response) {
```

Comment:
rename to createFileDownloadResponse - change signature to receive file - instead of generic object